packages/shared/src/libs/modules/socket/libs/enums/socket-event.enum.ts

```diff
@@ -3,6 +3,9 @@ const SocketEvent = {
 	CHAT_JOIN_ROOM: "chat-join-room",
 	CHAT_LEAVE_ROOM: "chat-leave-room",
 	CONNECTION: "connection",
+	NOTIFICATIONS_JOIN_ROOM: "notifications-join-room",
+	NOTIFICATIONS_LEAVE_ROOM: "notifications-leave-room",
+	NOTIFICATIONS_NEW_FOLLOWER: "notifications-new-follower",
```

Comment:
```suggestion
	NEW_NOTIFICATION: "new-notification",
```
I think there will be only one socket message handler for all type of notifications, so I think this enum value would be better